internal/cli/datalakepipelines/runs/list.go

```diff
@@ -59,17 +59,17 @@ func (opts *ListOpts) Run() error {
 	return opts.Print(r)
 }
 
-// atlas dataLakePipelines runs list [--projectId projectId].
+// ListBuilder represents atlas dataLakePipelines runs list [--projectId projectId].
 func ListBuilder() *cobra.Command {
 	opts := &ListOpts{}
 	cmd := &cobra.Command{
-		Use:     "list",
+		Use:     "list --pipeline pipeline",
```

Comment:
we historically have never documented options here

Reply:
Yes, I understand. This was the general feedback from customers, though. I see value in including it for required options in the list commands, especially since they typically don’t have any required options. WDYT?

Reply:
are you planning to backfill all commands with the missing required options?

Reply:
only the list commands if we all agree on this

Reply:
why the list commands and not the others?

Reply:
the main reason is that the `list` commands are (usually) the ones that don't require flags so it is easy for the user to forget about them.

Reply:
Hi @mmarcon 👋 Do you have any thoughts on using required flags for list commands in the command syntax? Should we apply this new format to all commands or only to list commands? Or should we use it at all?

Customers have provided feedback that required flags for list commands are not displayed in the syntax section. [Example](https://www.mongodb.com/docs/atlas/cli/current/command/atlas-accessLogs-list/#syntax): 
![Screenshot 2024-09-25 at 10 22 06](https://github.com/user-attachments/assets/e8519e68-da9c-4334-ac67-3ad6b749bf39)


Even though the `atlas accessLogs list` command requires a clusterName or hostname flag, they are not shown in the syntax section of the documentation.

This PR changes that section to be 
```
Usage:
  atlas accessLogs list {--clusterName clusterName | --hostname hostname} [flags]
```

Reply:
I'm goign to add here that if we decide to document "some" flags we shoudl at least follow http://docopt.org/

meaning it should be 
```
atlas accessLogs list (--clusterName <clusterName> | --hostname <hostname>) [options]
```

Reply:
@andreaangiolillo not sure you are going to be happy with my answer 😂

My perspective is that we should aim for consistency, i.e. if we believe a good user experience means including the required flags in the usage (which is something I can get behind, given that before working full time on the CLI understanding what flags were necessary without having to check the docs was something I struggled with too), then we should do it everywhere.

Having said that, I don't know if, with auto-generation coming, going through all the commands and adding the required flags manually is a good use of y'all's time. We can make sure that we account for displaying required flags as part of the auto-generation implementation.

@dianchenghu do you feel differently?

Reply:
@mmarcon I agree that the entire consistency issue should be reviewed at once (this PR could close one ticket, but we're aware there will be a bigger one still open in the backlog). For now, since the user particularly mentioned the accessLogs list example (and I believe it's not just a single case, list command is often used by human). I believe what @gssbzn  proposed here is not adding more confusion and it's a nice quick fix to have as soon as possible.